packages/react/src/lib/accessibility/Behaviors/Tooltip/tooltipAsLabelBehavior.ts

```diff
@@ -0,0 +1,39 @@
+import { Accessibility } from '../../types'
+import tooltipBehavior, { TooltipBehaviorProps } from './tooltipBehavior'
+
+/**
+ * @specification
+ * Adds attribute 'role=tooltip' to 'tooltip' slot.
+ * Adds attribute 'aria-hidden=false' to 'tooltip' slot if 'open' property is true. Sets the attribute to 'true' otherwise.
+ * Adds attribute 'aria-labelledby' based on the property 'aria-labelledby' to 'trigger' slot.
+ * Triggers 'close' action with 'Escape' on 'trigger'.
+ */
+const tooltipAsLabelBehavior: Accessibility<TooltipBehaviorProps> = props => {
+  const behaviorData = tooltipBehavior(props)
+  const defaultAriaLabeledBy = getDefaultAriaLabelledBy(props)
+
+  behaviorData.attributes = {
+    trigger: {
+      'aria-labelledby': defaultAriaLabeledBy || props['aria-labelledby'],
```

Comment:
do we want to add props['aria-labelledby'] ?, if we do already this check(verification) in "getDefaultAriaLabelledBy" function? :)